package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "realtime-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.12.0",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "node-sass": "^6.0.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
```

Comment:
### FYI 
`^x.x.x` you see here, means the project needs minimum of `x.x.x` version. 

For Example: `"react": "^17.0.2"` needs a react package of minimum version `17.0.2`, it would also work with `17.0.3` or `17.0.3`. But inorder to make sure everyone uses same package the specific version are mentioned in `package-lock.json`